packages/docs/components/navigation/components-navigation.js

```diff
@@ -0,0 +1,106 @@
+import data from './components-navigation.json';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { Nav, NavExpandable, NavGroup, NavItem, NavList } from '@patternfly/react-core';
+import classnames from 'classnames';
+import { createUseStyles } from 'react-jss';
+
+const useStyles = createUseStyles({
+    capitalize: {
+        textTransform: 'capitalize',
+        '& > button': {
+            textTransform: 'capitalize'
+        }
+    },
+    groupLink: {
+        'font-size': 14
+    }
+});
+
+const NavigationGroup = ({ group, items, packageName }) => {
+    const classes = useStyles();
+    const { pathname } = useRouter();
+    if (items.length === 1) {
+        const  item = items[0];
+        const title = typeof item === 'object' ? item.title : item;
+        const name = typeof item === 'object' ? item.name : item;
+        return (
+            <NavItem
+                id={`/fec/${packageName}/${name}`}
+                to={`/fec/${packageName}/${name}`}
+                ouiaId={`/fec/${packageName}/${name}`}
+                key={name}
+                component={({ children, ...props }) => (
+                    <Link {...props}>
+                        <a className={classnames('pf-c-nav__link', {
+                            'pf-m-current': props.href === pathname
+                        })}>
+                            {children}
+                        </a>
+                    </Link>
+                )}
+            >
+                {title}
+            </NavItem>
```

Comment:
This is fairly the same as is lines 55-76. Might want to be good to make this into a component